pkg/ui/changelog/changelog.go

```diff
@@ -0,0 +1,58 @@
+package changelog
+
+import (
+	"errors"
+
+	tea "github.com/charmbracelet/bubbletea"
+	changelog "github.com/futurice/jalapeno/pkg/ui/changelog/prompt"
+)
+
+func RunChangelog() ([]string, error) {
```

Comment:
Instead of return value of `[]string`, it could return a struct so [on this row](https://github.com/futurice/jalapeno/pull/140/files#diff-dc1f6e1c37681177c25d2a8b14e77a07b9a78c411d3cd0546a38006a9e6ed1aaR80) we don't need to implicitly pick the correct elements from the slice.